### ISSUE TITLE
Death Knight: Riding skill

### ISSUE THREAD

**Issue by Nienz** [Valhalla-Gaming/Tracker#6830]
**Describe the issue you're having**:
When i train riding skills and if i relog / server crash the riding skills are gone / not working

**Explain how you expect it work**:
Learn riding skill and it stays

**Steps to reproduce the problem**:

1. Buy riding skill on death knight ex: Flightmaster License
2. Relog
3. Check if you can use the proper mount ex: Flightmaster License and fly in Azeorth

**Links to Wowhead, YouTube, etc**:

**Comment by kjelt** [comment]
i can confirm this issue, playing a dk too. If you have luck only Expert Riding skill is not usable otherwise no riding skill is working.

**Comment by alltobe** [comment]
yeah me to since legion realise. this sucks :D becouse i have already explored these ares where can i fly :D
